src/coord/src/catalog/builtin.rs

```diff
@@ -563,6 +563,34 @@ FROM mz_catalog.mz_schemas",
     id: GlobalId::System(3012),
 };
 
+// TODO(jldlaughlin): add other fields - https://www.postgresql.org/docs/12/catalog-pg-class.html
+pub const PG_CLASS: BuiltinView = BuiltinView {
+    name: "pg_class",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_class AS SELECT
+oid,
+relname,
+NULL::oid as relowner
+FROM
+  (
+    SELECT oid, tables AS relname
+    FROM mz_catalog.mz_tables
+    UNION SELECT oid, sources AS relname
+    FROM mz_catalog.mz_sources
+    UNION SELECT oid, sinks AS relname
+    FROM mz_catalog.mz_sinks
+    UNION SELECT oid, views AS relname
+    FROM mz_catalog.mz_views
+    UNION (SELECT oid, name AS relname
+           FROM mz_catalog.mz_indexes
+           JOIN mz_catalog.mz_catalog_names
+           ON mz_catalog.mz_indexes.global_id = mz_catalog.mz_catalog_names.global_id
+           GROUP BY oid, name)
```

Comment:
Seems like `mz_indexes` should grow a `name` column! This is from memory, so could be wrong, but pretty sure the name columns elsewhere are (correctly) not qualified, while the names in `mz_catalog_names` are!

Reply:
yes, that's true! I think the complication has to do with what happens when the objects the index is on is renamed.. but I can look into it